charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```diff
@@ -10,39 +10,39 @@ data:
   cilium: |
     ChartName: cilium
     ChartVersion: 1.15.5
-    RepositoryURL: https://helm.cilium.io/
+    RepositoryURL: {{ if .Values.selfHostedRegistry }}oci://mindthegap.{{ .Release.Namespace }}.svc/charts{{ else }}https://helm.cilium.io/{{ end }}
```

Comment:
You can just use `oci://mindthegap` for the address, you don't need the namespace or svc suffix. This will resolve to that service in the same namespace as the pod.

Reply:
these were some extra steps probably from the trust manager version of this. i like this simplifications

Reply:
hmm now that i think about it- i think i need the namespace here or add the namespace in code because Repository URL is eventually going to be used by CAAPH. 

lets keep this explicit for right now. i appreciate the suggestion for trying to simplify this!